apps/ejabberd/src/ejabberd_local_delivery.erl

```diff
@@ -0,0 +1,34 @@
+%%%-------------------------------------------------------------------
```

Comment:
These comments aren't very useful right now. Please fill in the correct details (at least author email, possibly mongooseim-im@... if you don't want to provide your own email, company name and maybe one line description of the module) or drop these info comments completely.